MAINTAINERS

```diff
@@ -1,12 +1,11 @@
-Project Lead: @lukehinds
+Project Lead: @mpeters
 
 Maintainers:
 
 @lukehinds
 @jetwhiz
 @nabilschear
 @amylily1011
-@mpeters
```

Comment:
Just out of curiosity, is it correct to be removed from the list of maintainers? I noticed you were present in both before this change.

Reply:
Its neither correct / incorrect and has no bearing. I just thought it might be more tidy, but happy to go either way if you have an idea on how it could work.

Reply:
Nope, just curious